{"version":3,"file":"676.js","mappings":";;;;;;;;AAAa;AACb,8CAA6C;AAC7C;AACA,CAAC,EAAC;AACF,kBAAe;AACf,sBAAsB;AACtB,qBAAqB,mBAAO,CAAC,IAA+B;AAC5D;AACA;AACA;AACA;AACA;AACA;AACA;AACA,QAAQ,KAAsC,EAAE,EAS3C;AACL;AACA;;AAEA","sources":["webpack://closet/./node_modules/next/dist/lib/is-error.js"],"sourcesContent":["\"use strict\";\nObject.defineProperty(exports, \"__esModule\", {\n    value: true\n});\nexports.default = isError;\nexports.getProperError = getProperError;\nvar _isPlainObject = require(\"../shared/lib/is-plain-object\");\nfunction isError(err) {\n    return typeof err === \"object\" && err !== null && \"name\" in err && \"message\" in err;\n}\nfunction getProperError(err) {\n    if (isError(err)) {\n        return err;\n    }\n    if (process.env.NODE_ENV === \"development\") {\n        // provide better error for case where `throw undefined`\n        // is called in development\n        if (typeof err === \"undefined\") {\n            return new Error(\"An undefined error was thrown, \" + \"see here for more info: https://nextjs.org/docs/messages/threw-undefined\");\n        }\n        if (err === null) {\n            return new Error(\"A null error was thrown, \" + \"see here for more info: https://nextjs.org/docs/messages/threw-undefined\");\n        }\n    }\n    return new Error((0, _isPlainObject).isPlainObject(err) ? JSON.stringify(err) : err + \"\");\n}\n\n//# sourceMappingURL=is-error.js.map"],"names":[],"sourceRoot":""}